src/verifytime.c

```diff
@@ -32,17 +33,50 @@
 
 #define DAY_SECONDS 86400
 
+static char *path_join(const char *fmt, ...)
```

Comment:
add a comment here saying that this is a copy.

Reply:
ok